packages/erc3k/contracts/ERC3000Data.sol

```diff
@@ -44,6 +48,7 @@ library ERC3000Data {
     }
 
     function containerHash(bytes32 payloadHash, bytes32 configHash) internal view returns (bytes32) {
+        // Note: Split ERC name from version name
```

Comment:
What do you mean? It is gonna be packed into the same blob that will get hashed anyway